core/embed/extmod/modtrezorui/buffers.h

```diff
@@ -23,7 +23,7 @@
 #include <stdbool.h>
 
 #include "common.h"
-#include "display_defs.h"
+#include TREZOR_BOARD
```

Comment:
could we do `#include #TREZOR_BOARD` to avoid the awkward quoting from the def side?

Reply:
(otoh `MP_CONFIGFILE` also has quotes and it's easier to use it that way so i guess we'll keep it)